src/main/java/com/patternpedia/api/rest/controller/AuthorController.java

```diff
@@ -0,0 +1,98 @@
+package com.patternpedia.api.rest.controller;
+
+import com.patternpedia.api.entities.shared.AuthorConstant;
+import com.patternpedia.api.rest.model.candidate.CandidateModel;
+import com.patternpedia.api.rest.model.issue.IssueModel;
+import com.patternpedia.api.rest.model.shared.AuthorModel;
+import com.patternpedia.api.rest.model.shared.AuthorModelRequest;
+import com.patternpedia.api.rest.model.user.UserModel;
+import com.patternpedia.api.service.AuthorService;
+import com.patternpedia.api.service.UserService;
+import org.springframework.hateoas.CollectionModel;
+import org.springframework.hateoas.EntityModel;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping(value = "/authors", produces = "application/hal+json")
+public class AuthorController {
+
+    private AuthorService authorService;
+    private UserService userService;
+
+    public AuthorController(
+            AuthorService authorService,
+            UserService userService
+    ) {
+        this.authorService = authorService;
+        this.userService = userService;
+    }
+
+    /**
+     * GET Methods
+     */
+    @GetMapping(value = "")
+//    @PreAuthorize(value = "hasAuthority('ADMIN')")
```

Comment:
should these lines also be commented out if no authentication is used or is this line of code already obsolete?